src/components/AddTodoForm.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../App.css';
+
+export class AddTodoForm extends React.Component {
+  state = {
+    title: '',
+    userId: '',
+    hasTitleError: false,
+    hasUserIdError: false,
+  }
+
+  handleInputChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
+      hasTitleError: false,
+    });
+  }
+
+  handleSelectChange = (event) => {
+    const { value, name } = event.target;
+
+    this.setState({
+      [name]: Number(value),
+      hasUserIdError: false,
+    });
+  }
+
+  handleSubmit= (event) => {
+    event.preventDefault();
+
+    const { title, userId } = this.state;
+    const { addTodo } = this.props;
+
+    if (!title) {
+      this.setState({
+        hasTitleError: true,
+      });
+    }
+
+    if (!userId) {
+      this.setState({
+        hasUserIdError: true,
+      });
+    }
+
+    if (title && userId) {
+      this.setState({
+        hasTitleError: false,
+        hasUserIdError: false,
+      });
+
+      addTodo(title, userId);
+      this.setState({
+        userId: '',
+        title: '',
+      });
+    }
+  }
+
+  render() {
+    const { users } = this.props;
+    const { title, hasTitleError, hasUserIdError } = this.state;
+
+    return (
+      <>
+        <form
+          onSubmit={this.handleSubmit}
+          className="form"
+        >
+          <div className="error">
+            {
+            hasTitleError
+            && (
+            <div className="error__message">
+              Enter your title, please!
+            </div>
+            )
+            }
+
+            {
+            hasUserIdError
+            && (
+            <div className="error__message">
+              Choose user, please!
+            </div>
+            )
+            }
```

Comment:
Щось з форматуванням не так
```suggestion
            { hasTitleError && (
              <div className="error__message">
                Enter your title, please!
              </div>)}

            {hasUserIdError && (
              <div className="error__message">
                Choose user, please!
              </div>
            )}
```